test/jaxp/javax/xml/jaxp/libs/jaxp/library/JAXPTestUtilities.java

```diff
@@ -60,6 +61,15 @@
  * This is an interface provide basic support for JAXP functional test.
  */
 public class JAXPTestUtilities {
+    public static String CLS_DIR = System.getProperty("test.classes");
+    public static String SRC_DIR = System.getProperty("test.src");
+    public static boolean isWindows = false;
+    static {
+        if (System.getProperty("os.name").contains("Windows")) {
+            isWindows = true;
+        }
+    };
```

Comment:
Could be a direct assignment and final:
```suggestion
    public static final boolean isWindows = System.getProperty("os.name").contains("Windows");
```